test/e2e-upgrade-tests.sh

```diff
@@ -31,9 +31,16 @@
 
 source $(dirname $0)/e2e-common.sh
 
+latest_version() {
```

Comment:
test infra actually supports this - https://github.com/knative/test-infra/blob/master/scripts/library.sh#L747-L772, it's looking for the latest release from GCS, which is less ideal than `git tag` though.

Reply:
This is a pure cherry-pick.  If we have something better, want to send a PR to HEAD?